src/types/ErrorsMessages.ts

```diff
@@ -0,0 +1,7 @@
+export enum ErrorsMessages {
+  Load = 'Unable to load a todos',
```

Comment:
Consider using more descriptive names for error messages. For example, instead of `Load`, use `UnableToLoadTodos`. 📝